definitions/pipelines/rnaseq.cwl

```diff
@@ -52,8 +52,11 @@ inputs:
 outputs:
     final_bam:
         type: File
-        outputSource: index_bam/indexed_bam
+        outputSource: mark_dup/sorted_bam
         secondaryFiles: [.bai]
+    markdup_metrics:
```

Comment:
I guess it doesn't hurt to output these metrics, but it's probably not really needed!

Reply:
removed that as an output.